src/main/java/net/sf/jabref/logic/groups/KeywordGroup.java

```diff
@@ -370,7 +371,7 @@ public static String getDescriptionForPreview(String field, String expr, boolean
     public String getShortDescription() {
         StringBuilder sb = new StringBuilder();
         sb.append("<b>");
-        if (Globals.prefs.getBoolean(JabRefPreferences.GROUP_SHOW_DYNAMIC)) {
+        if (jabRefPreferences.getBoolean(JabRefPreferences.GROUP_SHOW_DYNAMIC)) {
```

Comment:
is it possible to pass `jabRefPreferences.getBoolean(JabRefPreferences.GROUP_SHOW_DYNAMIC)` as a boolean parameter to getShortDescription?